src/components/CheckboxField/CheckboxField.tsx

```diff
@@ -47,10 +45,6 @@ export interface Props {
    * HTML label text
    */
   label?: string;
-  /**
-   * Slot for node to appear directly after field legend. Typically used to include a Toolip
-   */
-  legendAfter?: ReactNode;
```

Comment:
I wonder what this use case actually looked like. Wish they had an example somewhere.

Reply:
It was [appending this after](https://github.com/chanzuckerberg/edu-design-system/pull/971/files#diff-070bda2885e117424aa887bf9acfd459bd90ae98e74213a05058371eeeba02a8L68-L70) the `optionalLabel` and was probably used for elements such as a Tooltip as suggested above. For our tooltip at least we can pass the ref, add a trigger child element within the `Fieldset` wrapper, or any other approach that our Tooltip allows